exporter.php

```diff
@@ -1,257 +1,42 @@
-}
-
-/**
- * Get the Post Tag description.
- *
- * @param object $tag Tag object.
- */
-function p4_px_single_post_tag_description( $tag ) {
-	if ( empty( $tag->description ) ) {
-		return;
-	}
-
-	echo '<wp:tag_description>' . p4_px_single_post_cdata( $tag->description ) . '</wp:tag_description>';
-}
-
-/**
- * Get the Post Term name.
- *
- * @param object $term Term object.
- */
-function p4_px_single_post_term_name( $term ) {
-	if ( empty( $term->name ) ) {
-		return;
-	}
-
-	echo '<wp:term_name>' . p4_px_single_post_cdata( $term->name ) . '</wp:term_name>';
-}
-
-/**
- * Get the Post Term description.
- *
- * @param object $term Tag object.
- */
-function p4_px_single_post_term_description( $term ) {
-	if ( empty( $term->description ) ) {
-		return;
-	}
-
-	echo '<wp:term_description>' . p4_px_single_post_cdata( $term->description ) . '</wp:term_description>';
-}
-
-/**
- * Get the Post's authors.
- *
- * @param array $post_ids Tag object.
- */
-function p4_px_single_post_authors_list( $post_ids ) {
-	global $wpdb;
-
-	$post_ids = array_map( 'intval', $post_ids );  // santize the post_ids manually.
-	$post_ids = array_filter( $post_ids ); // strip ones that didn't validate.
-
-	$authors = [];
-
-	$placeholders = implode( ',', array_fill( 0, count( $post_ids ), '%d' ) );
-
-	$sql = 'SELECT DISTINCT post_author
-			FROM %1$s
-			WHERE ID IN(' . $placeholders . ') AND post_status != \'auto-draft\'';
-
-	$values       = [];
-	$values[0]    = $wpdb->posts;
-	$values       = array_merge( $values, $post_ids );
-	$prepared_sql = $wpdb->prepare( $sql, $values );     // WPCS: unprepared SQL OK.
-	$results      = $wpdb->get_results( $prepared_sql ); // phpcs:ignore
-
-	foreach ( (array) $results as $result ) {
-		$authors[] = get_userdata( $result->post_author );
-	}
-
-	$authors = array_filter( $authors );
-
-	foreach ( $authors as $author ) {
-		echo "\t<wp:author>";
-		echo '<wp:author_id>' . $author->ID . '</wp:author_id>';
-		echo '<wp:author_login>' . $author->user_login . '</wp:author_login>';
-		echo '<wp:author_email>' . $author->user_email . '</wp:author_email>';
-		echo '<wp:author_display_name>' . p4_px_single_post_cdata( $author->display_name ) . '</wp:author_display_name>';
-		echo '<wp:author_first_name>' . p4_px_single_post_cdata( $author->user_firstname ) . '</wp:author_first_name>';
-		echo '<wp:author_last_name>' . p4_px_single_post_cdata( $author->user_lastname ) . '</wp:author_last_name>';
-		echo "</wp:author>\n";
-	}
-}
-
-/**
- * Get the Post Taxonomy.
- */
-function p4_px_single_post_taxonomy() {
-	$post = get_post();
-
-	$taxonomies = get_object_taxonomies( $post->post_type );
-	if ( empty( $taxonomies ) ) {
-		return;
-	}
-	$terms = wp_get_object_terms( $post->ID, $taxonomies );
-
-	foreach ( (array) $terms as $term ) {
-		echo "\t\t<category domain=\"{$term->taxonomy}\" nicename=\"{$term->slug}\">" . p4_px_single_post_cdata( $term->name ) . "</category>\n";
-	}
-}
```

Comment:
Nice, moving the these functions out of this file really makes it a lot more manageable :100: